android/src/facebook/TiFacebookModule.java

```diff
@@ -259,11 +259,17 @@ public void onCompleted(GraphResponse response) {
 	}
 	
 	@Kroll.method
-	public void logCustomEvent(String event) {
+	public void logCustomEvent(String event, @Kroll.argument(optional = true) Double valueToSum, @Kroll.argument(optional = true) KrollDict parameters) {
 		Activity activity = TiApplication.getInstance().getCurrentActivity();
 		AppEventsLogger logger = AppEventsLogger.newLogger(activity);
+		Bundle paramBundle = parameters != null ? Utils.mapToBundle(parameters) : null;
+		Log.d(TAG, "logCustomEvent: "+ event + " valueToSum: " + valueToSum);
 		if (logger != null) {
-			logger.logEvent(event);
+			if (valueToSum == null){
+				logger.logEvent(event, paramBundle);
+			}else{
```

Comment:
Missing spaces in `if (){` and `}else{`. Also please remove the debug event for now. Thanks!